capstone/src/test/javascript/graphsTest.js

```diff
@@ -0,0 +1,17 @@
+describe('Data Table Construction', function() {
+  it('Will add correct columns/rows to DataTable from JSON', function() {
+    // TODO(ihsan314): Change this when format of JSON object is well-defined
```

Comment:
What does `this` mean in the `TODO`?

Reply:
The `TODO` makes more sense with the definition of the `JSON` object and the `DataTables`. I'll move it there.